app/locale/en.js

```diff
@@ -3327,6 +3327,7 @@ module.exports = {
       difficulty_beginner: 'Beginner',
       difficulty_intermediate: 'Intermediate',
       difficulty_advanced: 'Advanced',
+      difficulty_unknown: '', // keep empty string
```

Comment:
why do we need this? Why not show `-`?

Reply:
i add this because seems GD team doesn't set the difficulty correct (my fault) so the difficulty shows as unknown. so add an empty to get rid of `unknown` for now so we have time to change level db

Reply:
let's shown Unknown for now. When we cleanup the data, it will be fixed. Let's not add strings to en.js unless we need it.